benchmarks/src/main/scala/zio/IOLeftBindBenchmark.scala

```diff
@@ -120,11 +120,11 @@ class IOLeftBindBenchmark {
 
   private[this] def zioLeftBindBenchmark(runtime: Runtime[Any]): Int = {
     def loop(i: Int): UIO[Int] =
-      if (i % depth == 0) IO.succeedLazy[Int](i + 1).flatMap(loop)
-      else if (i < size) loop(i + 1).flatMap(i => IO.succeedLazy(i))
-      else IO.succeedLazy(i)
+      if (i % depth == 0) IO.succeed[Int](i + 1).flatMap(loop)
```

Comment:
This should become `effectTotal` in line with other effect types that delay execution in their benchmarks (e.g. monix `Task.eval`)

Reply:
:heavy_check_mark: